modules/apps/commerce/commerce-frontend-js/test/dev/public/js/static-env-utils.js

```diff
@@ -45,11 +45,22 @@ window.Liferay = {
 	on: (name, fn) => {
 		window.addEventListener(name, fn);
 	},
-	staticEnvHeaders: new Headers({
+};
+
+window.defaultFetch = fetch;
+
+window.fetch = (resource, {headers, ...init}) => {
```

Comment:
Couldn't you use [`fetch` from `frontend-js-web`](https://github.com/liferay/liferay-portal/blob/cb2b504c40d7a086dd7fa18f183b016ce2dd2347/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/fetch.es.js) and pass those `Headers`?

Reply:
Julien on this one I preferred keeping it as it is. This file contains some utilities we place a static env to replace standard liferay ones (like `Liferay.Language.get` or `themeDisplay.getLanguageId`) and it doesn't get compiled. 

Its only purpose is avoiding to write exceptions (that are aimed to make everything working in static env) in code which will be deployed